Source/Private/ASDisplayNodeInternal.h

```diff
@@ -75,6 +75,7 @@ AS_EXTERN NSString * const ASRenderingEngineDidDisplayNodesScheduledBeforeTimest
 #define TIME_DISPLAYNODE_OPS 0 // If you're using this information frequently, try: (DEBUG || PROFILE)
 
 #define NUM_CLIP_CORNER_LAYERS 4
+#define MAX_INTERFACE_STATE_DELEGATES 4
```

Comment:
Most nodes don't need more than 4 delegates, but there should be an API to increase it. That way users can do so without changing the framework.